examples/toggle_led/README.md

```diff
@@ -1,75 +1,77 @@
 # Astarte LED Toggle example
 
-## Usage
-First of all, you need an Astarte instance to send data to.
+This example will guide you through setting up a simple ESP32 application running the Astarte SDK
+and interacting with a local instance of Astarte.
 
-If you don't have one, you can either follow the [Astarte in 5
-minutes](https://docs.astarte-platform.org/latest/010-astarte_in_5_minutes.html) guide or deploy it
-to you favorite cloud provider using our [Kubernetes
-Operator](https://github.com/astarte-platform/astarte-kubernetes-operator).
+We will configure the device to perform the following actions:
+- Auto-register in the local Astarte instance at the first start-up.
+- When a message is received from our local instance of Astarte blinks an onboard LED.
```

Comment:
- Blink an onboard LED when a message is received from our local instance of Astarte;